rust/gui-client/README.md

```diff
@@ -75,6 +75,23 @@ If that doesn't work:
 - Click `New client secret`
 - Note down the secret value. This should be entered into the GitHub repository's secrets as `AZURE_CLIENT_SECRET`.
 
+## Installing on CentOS 9
+
+These instructions will move to the knowledge base once the first release supporting CentOS 9 is cut.
+
+1. Download RPM
+2. `sudo dnf install systemd-resolved` (Installing it explicitly prevents it from being auto-removed if Firezone is removed)
+3. `sudo dnf install epel-release` (Needed to get GNOME extensions)
+4. `sudo dnf install ./firezone-client-gui-*.rpm`
+5. `sudo usermod -aG firezone-client $USER`
+6. `sudo systemctl enable firezone-client-ipc.service` (See https://www.freedesktop.org/software/systemd/man/latest/systemd.preset.html, "It is not recommended to ship preset files within the respective software packages implementing the units")
+7. `sudo systemctl enable systemd-resolved.service` (In addition to the preset thing, the family of Fedora distros seem to have a policy that installing a service should never enable or auto-start it. This might be contrary to Debian-family.)
+8. `sudo dnf install gnome-shell-extension-appindicator` (Install the system tray)
+9. Reboot to add user to group fully, start new services, and make the appindicator extension available.
+10. `sudo ln --force --symbolic /run/systemd/resolve/stub-resolv.conf /etc/resolv.conf`
```

Comment:
Is that because we try to read resolv.conf and it isn't there otherwise on these distros?

Reply:
If I do another test run I'll double-check, but I think systemd-resolved just doesn't try to control this file. As admin we have to switch it over

Reply:
Looks like systemd-resolved does not try to control it even when enabled and started. I noted in the docs that it's probably because it's under `/etc` so it would be against policy for a package to automatically change it. 🤷‍♀️